x-pack/plugins/alerting_builtins/server/alert_types/geo_threshold/es_query_builder.ts

```diff
@@ -18,11 +18,13 @@ export async function getShapesFilters(
   boundaryType: string,
   boundaryGeoField: string,
   geoField: string,
+  boundaryNameField?: string,
   callCluster: ILegacyScopedClusterClient['callAsCurrentUser'],
   log: Logger,
   alertId: string
 ) {
   const filters: Record<string, unknown> = {};
+  const shapesIdsNamesMap = {};
```

Comment:
Couldn't use an actual JS `Map` here on account that this variable gets serialized when assigned to `state`.